http_check/check.py

```diff
@@ -169,7 +169,11 @@ def _load_conf(self, instance):
         http_response_status_code = str(instance.get('http_response_status_code', DEFAULT_EXPECTED_CODE))
         timeout = int(instance.get('timeout', 10))
         config_headers = instance.get('headers', {})
-        headers = agent_headers(self.agentConfig)
+        default_headers = instance.get("include_default_headers", True)
+        if default_headers:
+            headers = agent_headers(self.agentConfig)
+        else:
+            headers={}
```

Comment:
I will add these changes (whitespace and boolean check) as soon as I can, thx for the feedback.